lib/zendesk_apps_tools/deploy.rb

```diff
@@ -87,22 +100,21 @@ def check_status(response, poll_job = true)
     end
 
     def check_job(job_id)
-      connection = get_connection
+      check_job_connection = connection
```

Comment:
Hmm.. probably not necessary. The reason why `connection` is assigned to `check_job_connection` here, is because of the loop that follows (line 105). 

`connection` which is essentially `get_connection` (from APIConnection Module) goes through the process of preparing network client and fetching credentials / prompting for user inputs. I didn't to want to spam the process in the loop.

as for using `alias_method :connection, :get_connection`,
it provides a DRYER option where we don't have to repeat `connection = get_connection` in all the methods of this Module.

Reply:
When discuss in real life, we came up with a more appropriate name for variable `check_job_connection`. We should name it what connection represents before the loop block. I will rename this as `network_client`